spec/lib/telegram/bot/client_spec.rb

```diff
@@ -0,0 +1,17 @@
+RSpec.describe Telegram::Bot::Client do
+  let(:token) { '180956132:AAHU0_CeyQWOd6baBc9TibTPybxY9p1P8xo' }
+
+  describe '#new' do
```

Comment:
What about using more `let`? I think it is more natural for specs than pure assingments
```
RSpec.describe Telegram::Bot::Client do
  let(:token)  { '180956132:AAHU0_CeyQWOd6baBc9TibTPybxY9p1P8xo' }
  let(:client) { described_class.new(token, options) }

  describe '#new' do
    subject { client.options }
    
    context 'with any options' do
      let(:options) { { offset: 1, timeout: 30, something: true } }
    
      it 'accepts any option' do
        is_expected.to eql(options)
      end
    end

    context 'with logger option' do
      let(:options) { { offset: 1, timeout: 30, something: true, logger: Logger.new('/dev/null') } }
      
      it 'removes logger from options' do
        is_expected.not_to include(:logger)
      end
    end
  end
end
```